packages/design-system/src/components/DateField/DateField.tsx

```diff
@@ -23,14 +22,16 @@ export interface DateFieldProps {
    * Additional classes to be added to the root fieldset element
    */
   className?: string;
+  disabled?: boolean;
```

Comment:
Good pickup on `disabled` field as it's the react file but was missing from the original typedef file.
The `disabled` line can be moved after the `dateFormatter` prop.

Reply:
`lableId` is also on the react file, but was missing from the original typedef file - need to be added here too.